### DISCUSSION TITLE
LFS file un/locking

### DISCUSSION THREAD

**Discussion by erdalkaraca** [eclipse-jgit/jgit#50] [General]
Is file un/locking something already discussed? What would be required to implement it (without depending on external CLI tools)?

**Comment by erdalkaraca** [comment]
@msohn any ideas/hints?

**Comment by msohn** [comment]
AFAIK lfs locking is not implemented in jgit lfs. 
We don't use lfs hence my understanding of how git lfs works today is probably outdated.
